src/state/state-display-mixin.ts

```diff
@@ -31,6 +32,10 @@ export default <T extends Constructor<HassBaseEl>>(superClass: T) => {
 
     private _updateStateDisplay = async () => {
       if (!this.hass) return;
+
+      const { numeric_device_classes: sensorNumericDeviceClasses } =
+        await getSensorNumericDeviceClasses(this.hass);
```

Comment:
Would it make any sense to just put this list in `hass`, so we don't have to deal with the async function in each module that use this information?

Reply:
It already is async now right?

Reply:
> Do we have to load this list every time the entity reg is updated?

I don't think we do. Are you suggesting I'm loading it too often? I think we only need to load it one time ever, and it never changes after that. Believe that's why the result is cached in data/sensor.